dask/base.py

```diff
@@ -498,18 +481,24 @@ def persist(*args, **kwargs):
 
     try:
         from distributed.client import default_client
-        collections = default_client().persist(collections, **kwargs)
-        if isinstance(collections, list):  # distributed is inconsistent here
-            collections = tuple(collections)
+    except ImportError:
+        pass
+    else:
+        try:
+            client = default_client()
+        except ValueError:
```

Comment:
This is fine, but going forward you might consider refactoring `default_client` to either return a sentinel value (i.e., `None`) if there is no default or raise a sentinel specific error, e.g., `NoDefaultClientError`.